srcpkgs/ocs-url/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'ocs-url'
+pkgname=ocs-url
+version=3.1.0
+revision=1
+#archs="x86_64"
+wrksrc=$pkgname-release-$version
+build_style=qmake
+hostmakedepends="qt5-qmake xdg-utils git"
+makedepends="qt5-devel qt5-svg-devel qt5-declarative-devel qt5-quickcontrols2-devel"
+short_desc="Install helper program for items served via (ocs://)"
+maintainer="Guilherme Machado <machadofguilherme@gmail.com>"
+license="GPL-3.0"
```

Comment:
Needs to be either GPL-3.0-only or GPL-3.0-or-later

Reply:
> Needs to be either GPL-3.0-only or GPL-3.0-or-later

License fixed. =]